Babylon/groups/powerbi/groups/workspace/commands/get.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logging
+import pathlib
+from typing import Optional
+
+import requests
+from click import Context
+from click import Path
+from click import command
+from click import option
+from click import pass_context
+
+from ......utils.decorators import require_deployment_key
+from ......utils.logging import table_repr
+from ......utils.typing import QueryType
+
+logger = logging.getLogger("Babylon")
+
+
+@command()
+@pass_context
+@option(
+    "-o",
+    "--output-file",
+    "output_file",
+    help="The path to the file where workspace should be outputted (json-formatted)",
+    type=Path(dir_okay=False, path_type=pathlib.Path),
+)
+@option("-w", "--workspace", "override_workspace_id", required=False, type=QueryType())
+@require_deployment_key("powerbi_workspace_id", required=False)
+def get(ctx: Context, powerbi_workspace_id: str, override_workspace_id: Optional[str], output_file: Optional[Path]):
+    """Get info about a specific workspace from Power BI"""
+    access_token = ctx.parent.obj.token
+    workspace_id = override_workspace_id or powerbi_workspace_id
+    url_groups = f'https://api.powerbi.com/v1.0/myorg/groups?$filter=contains(id,\'{workspace_id}\')'
+    header = {'Content-Type': 'application/json', 'Authorization': f'Bearer {access_token}'}
+    api_out = requests.get(url=url_groups, headers=header)
+    groups = api_out.json()['value']
```

Comment:
```suggestion
    groups = api_out.json().get("value")
```
I would use get because you are testing if it is None afterwards, here it would throw a KeyError